src/main.cc

```diff
@@ -367,6 +373,8 @@ std::vector<Option> options_
        "Allow the server to listen for GRPC requests."},
       {OPTION_GRPC_PORT, "grpc-port", Option::ArgInt,
        "The port for the server to listen on for GRPC requests."},
+      {OPTION_GRPC_ADDR, "grpc-addr", Option::ArgStr,
```

Comment:
nit: `xxx-address`

Reply:
Should I also make changes to all the parameters/arguments?